src/main/resources/db/changelog/changes/01-create-users-table.yaml

```diff
@@ -12,6 +12,11 @@ databaseChangeLog:
                   autoIncrement: true
                   constraints:
                     primaryKey: true
+              - column:
+                  name: chat_Id

```

Comment:
I think it's better to have consistent naming and the version in model looks better to me
```
@Column(name = "chat_id", unique = true)
private Long chatId;
```